authapi/tests.py

```diff
@@ -296,6 +371,7 @@ def test_serializer(self):
             'id': team.id,
             'users': [
                 UserSummarySerializer(instance=user, context=context).data],
```

Comment:
Do we need to check that inactive users aren't shown here? Same for organizations.